WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListActivity.kt

```diff
@@ -309,4 +335,13 @@ class PostsListActivity : AppCompatActivity(),
     override fun onDismissByOutsideTouch(instanceTag: String) {
         viewModel.onDismissByOutsideTouchForBasicDialog(instanceTag)
     }
+
+    // Menu PostListViewLayoutType handling
+
+    private fun updateMenuIconForViewLayoutType(@DrawableRes iconId: Int) {
+        getDrawable(iconId)?.let { drawable ->
+            drawable.setTint(Color.WHITE)
```

Comment:
I think we can remove this line now as the drawables are already white. If we decide not to remove it we might want to consider passing the tintColor from the view model. Wdyt?

Reply:
Done. No reason to have it anymore so I'll keep it super simple.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9657/commits/762018575fe953b599121659208dcf1ba5f80498